tests/requirements.txt

```
@@ -11,9 +11,10 @@ cffi==1.14.1
 chardet==3.0.4
 click==7.1.2
 colour==0.1.5
-cryptography==3.3.2
+cryptography<3.4;python_version<"3.7"
 cycler==0.10.0
 dash[testing,dev]
+importlib-metadata==4.8.3
```

```suggestion
importlib-metadata==4.8.3;python_version<"3.7"
```
That's what we have in [dash](https://github.com/plotly/dash/pull/1872) anyway